NoSoft-SA/nspack
# Feature/reassign staging bays

Summary of this change:
Functionality to reassign a load's staging bay via the load staging screen.
[Full spec here](https://docs.google.com/document/d/172ZDijuIoUKNIBsEo66MfHx2BarOaYG17v2bhLR-LGc/edit?usp=sharing)


CHANGELOG entry:

### Added
- `can_reassign_staging_bays` user permission, required to perform the changes in this pull request
- View to reassign the staging bay via the `reassign staging bay` button in the `load staging` screen.
- Column `previous_staging_location_id` to `loads`
- A list of pallets which belongs to the load, still at the previous staging location, to both the load and load staging screens.

How to test this code via the UI:
Finished goods | dispatch | list loads | (via action menu) load staging

Ensure the permission is set and that the appropriate type of load is trying to be reassigned to a new bay (otherwise "load staging" will not be accessible via the action menu).
